### PR TITLE
Integration icons updates 

### LINKED ISSUE #1760: [Website] Integrations icon updates
#### Current State of the Meshery.io site

Homepage: https://meshery.io/

Some application icons of integrations are missing and being default icon.

#### Desired State of the Meshery.io site

Review Integrations icons and updates.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by shubhusion**
**Description**

This PR fixes #1760

**Notes for Reviewers**

- Changed icons for apisix and gitlab.
- Added missing icons for wordpress-operator and aws-ecs.
- Some icons like GitHub, kubesphere, and Prometheus appear twice with the same permalinks in the integration icon grid on the homepage. Should I remove one of them?

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: catalog/0194ff83-0a43-4b83-9c75-b8f0f32da6b7/artifacthub-pkg.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: 0.0.1
 name: pod-readiness
 displayName: Pod Readiness
-createdAt: "2024-07-01T12:40:18Z"
+createdAt: "2024-07-02T00:24:53Z"
```
Comment (sudhanshutech) [inline catalog/0194ff83-0a43-4b83-9c75-b8f0f32da6b7/artifacthub-pkg.yml:4]: why this change?

File: catalog/28715e69-c6c1-4f96-bfa2-05113b00bae0/artifacthub-pkg.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: 0.0.1
 name: cryptomb-tls-handshake-acceleration-for-istio
 displayName: CryptoMB-TLS-handshake-acceleration-for-Istio
-createdAt: "2024-07-01T12:40:35Z"
+createdAt: "2024-07-02T00:25:25Z"
```
Comment (sudhanshutech) [inline catalog/28715e69-c6c1-4f96-bfa2-05113b00bae0/artifacthub-pkg.yml:4]: why?

**Comment by shubhusion** [comment]
I only made 2 commits. But here it shows 5. I guess i need to again rebase my branch and make a new PR

File: catalog/28715e69-c6c1-4f96-bfa2-05113b00bae0/artifacthub-pkg.yml (hunk 2 above)
Comment (shubhusion) [reply catalog/28715e69-c6c1-4f96-bfa2-05113b00bae0/artifacthub-pkg.yml:4]: I only made 2 commits. But here it shows 5. I guess i need to again rebase my branch and make a new PR
